Sources/TuistLoader/Loaders/CachedManifestLoader.swift

```diff
@@ -101,6 +100,13 @@ public class CachedManifestLoader: ManifestLoading {
     // MARK: - Private
 
     private func load<T: Codable>(manifest: Manifest, at path: AbsolutePath, loader: () throws -> T) throws -> T {
+        if case .config = manifest, cacheDirectory == nil {
+            return try loader()
+        } else if cacheDirectory == nil {
+            let config = try TuistGraph.Config.from(manifest: try loadConfig(at: path), at: path)
```

Comment:
we should run this only if the `manifest` is `.config`, shouldn't we? Right now this is run whenever `cacheDirectory` is nil

Reply:
The goal here is to load the config so we can set the cacheDirectory for the other type of manifests. Maybe if it is not clear I could try to extract as a lazy var? or do you have other suggestions? 🤔

Reply:
Actually, this part does not seem to be correct, as the path is the root of others manifest. Do you have ideas on how I can find where Config is located?

Reply:
yes, exactly!